src/components/login/Providers.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent } from 'react'
+import { LoginProvider } from '../../types/Login'
+import ProviderLink from './ProviderLink'
+import styles from './Providers.module.scss'
+
+interface Props {
+  providers: LoginProvider[],
+}
+
+const LoginProviders: FunctionComponent<Props> = ({
+  providers,
+}) => {
+  const links = providers.map(p => <ProviderLink provider={p} key={p.method} />)
```

Comment:
Seems CI doesn't like this `map` call, which surprises me because the type of the `providers` prop is an array :thinking: It definitely works at runtime

Reply:
Well, the pipeline tries to build this against the current backend (that's part of why I would have liked a mono repo) so this will call https://flathub.org/api/v2/auth/login as that's the endpoint that defines the providers. But as the endpoint doesn't exist (as the backend isn't merged) this variable will be empty.

Reply:
The backend is now merged.

Reply:
CI now passing :grin:

However, since the network request for providers could potentially fail at build time (the same as backend not being present) I've added some error handling.